kolibri/plugins/coach/assets/src/views/lessons/ManageLessonModals/DeleteLessonModal.vue

```diff
@@ -5,7 +5,10 @@
     @cancel="closeModal()"
   >
     <p>{{ $tr('lessonDeletionConfirmation', { title: lessonTitle }) }}</p>
-    <div class="core-modal-buttons">
+    <form
```

Comment:
I'm not sure this is really a form. If you want that form-like click-enter-to-submit behavior, you can register a `@enter` listener on core-modal.

Reply:
created an issue for this: https://github.com/learningequality/kolibri/issues/3311